melodic/distribution.yaml

```diff
@@ -12140,6 +12140,17 @@ repositories:
       url: https://github.com/uos/uos_tools.git
       version: master
     status: maintained
+  ur_client_library:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/UniversalRobots/Universal_Robots_Client_Library-release.git
+      version: 0.1.0-1
+    source:
```

Comment:
can you add a doc tab?

Reply:
That would require adding a `rosdoc.yaml` file and `mainpage.dox` file through the release repository as described [here](http://wiki.ros.org/bloom/Tutorials/ReleaseThirdParty#Modifying_the_Release_Repository), right?

I didn't want to clutter the source repository with that, as it is quite ROS specific. On the other hand, we have a `package.xml` file there, already...

Reply:
> That would require adding a rosdoc.yaml file and mainpage.dox file through the release repository as described here, right?

I think that's only required when you don't use doxygen (see [here](http://wiki.ros.org/rosdoc)), maybe @cottsay can chime in.

Reply:
Sorry, I have absolutely no experience with the doc generation...

Reply:
As I have already added a doc entry for the [Universal Robots ROS Driver](https://github.com/ros/rosdistro/blob/master/melodic/distribution.yaml#L12174) which created also documentation for contained packages, which don't have a respective `rosdoc.yaml` and `mainpage.dox` such as [ur_calibration](http://docs.ros.org/melodic/api/ur_calibration/html/) it should be fine adding a doc entry without explicitly setting this up.

Reply:
I've added a doc section to this PR.